integration-tests/scripts/find_release_pipelines_from_pr.sh

```diff
@@ -198,7 +198,9 @@ _find_and_process_pipelines() {
     export FOUND_PIPELINES="${FOUND_PIPELINENAMES[*]}"
   fi
 
-  ALL_TESTCASES=("e2e" "rh-advisories" "fbc-release" "release-to-github" "push-to-external-registry" "rhtap-service-push" "rh-push-to-registry-redhat-io" "rh-push-to-external-registry" "push-to-addons-registry")
+  ALL_TESTCASES=("e2e" "rh-advisories" "fbc-release" "release-to-github" "push-to-external-registry" \
+  "rhtap-service-push" "rh-push-to-registry-redhat-io" "rh-push-to-external-registry" "push-to-addons-registry" \
+  "push-disk-images-to-cdn" "push-disk-images-to-marketplaces", "push-rpms-to-pulp")
```

Comment:
we don't have a test for push-disk-images-to-cdn or push-disk-images-to-marketplaces